src/main/java/seedu/address/logic/commands/DeleteItemCommand.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ITEM_NAME;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.item.Item;
+import seedu.address.model.recipe.Recipe;
+
+/**
+ * DeleteItemCommand represents a delete item command with hidden
+ * internal logic and the ability to be executed.
+ */
+public class DeleteItemCommand extends Command {
+    public static final String COMMAND_WORD = "deli";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a item in the item list. "
+            + "Parameters: "
+            + PREFIX_ITEM_NAME + "ITEM NAME\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_ITEM_NAME + "Iron ";
+    public static final String MESSAGE_SUCCESS = "Item and Recipes of the item Deleted: %1$s";
+    public static final String MESSAGE_ITEM_NOT_FOUND = "Item is not found in the item list";
+    public static final String MESSAGE_ITEM_ALREADY_DELETED = "Item was already previously deleted";
```

Comment:
Should this use the NOT_FOUND message instead? User does not need to know our implementation